carbon-design-system/carbon-addons-iot-react
# feat(table): toolbar is not shown when only row actions are present

Closes #1423

**Summary**

Row actions no longer render a toolbar, also added if multiselect is present and there is a selection, the toolbar is shown.

**Change List (commits, features, bugs, etc)**

- Removes the check for row actions when rendering the toolbar, but adds in check for multiselect and selection

**Acceptance Test (how to verify the PR)**

- Verify that Table 'with row expansion and actions' no longer supplies an empty toolbar
- Verify that there are no other row actions that require toolbar support
